admin_manual/maintenance/migrating.rst

```diff
@@ -29,3 +29,112 @@ To start, let us be specific about the use case. A configured ownCloud instance
 
 #.  Change the ``CNAME`` entry in the DNS to point your users to the new
     location.
+
+
+
+=================
+Example migration
+=================
+
+.. note:: For this example to work, you need **ssh** on both servers and the **PermitRootLogin** to be set to "yes" on the new server.
+
+Install SSH::
+
+   apt install openssh-server openssh-client -y
+
+Edit SSH-Config::
+
+   nano /etc/ssh/sshd_config
+
+Change PermitRootLogin to yes::
+
+   PermitRootLogin yes
+
+=========
+Migration
+=========
+
+1. Install ownCloud on new server
+
+2. Put original server in maintenance mode:
+
+Go in owncloud dir::
+
+      cd /var/www/owncloud/
+
+
+Switch to maintenance mode::
+
+      sudo -u www-data php occ maintenance:mode --on
+
+
+wait for 6-7 min and stop apache2::
+
+   service apache2 stop
+
+3. Export and Import the database
+
+Go in owncloud dir::
+
+      cd /var/www/owncloud/
+
+Export on original server
+
+SYNOPSIS::
+
+   mysqldump --lock-tables -h [server] -u [username] -p[password] [db_name] > owncloud-dbbackup_`date +"%Y%m%d"`.bak
+
+Example::
+
+   mysqldump --lock-tables -h localhost -u admin -ppassword owncloud > owncloud-dbbackup.bak
+
+.. note:: Parameters are set when creating database for ownCloud
+
+::
+
+   CREATE DATABASE IF NOT EXISTS owncloud;
+   GRANT ALL PRIVILEGES ON owncloud.* TO 'username'@'localhost' IDENTIFIED BY 'password';
```

Comment:
Okay

 a.) CREATE DATABASE is not a command that is meant to be executed

https://github.com/owncloud/documentation/pull/2965/commits/53c0aced6ef9a3c563531ff3ef06f0a258b431bb

"

Note

(just a reminder: Parameters you need for the command above

were set when creating database for ownCloud with this command)"

"Do not execute this command, because it's just a reminder."

I added a few extra lines to clarify it.

b) I will implement it, thanks